syntax = "proto3";

message ExecutionResponse {
    string status = 1;
}

message DbAction {
    string type = 1;
    string table_name = 2;
    repeated string col_names = 3;
    string values_json = 4;
}

service Replication {
    rpc propogate( stream DbAction) returns (ExecutionResponse) {}
}